internal/services/managedhsm/client/helpers.go

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) HashiCorp, Inc.
```

Comment:
actually - why is this so different this https://github.com/hashicorp/terraform-provider-azurerm/blob/main/internal/services/keyvault/client/helpers.go ?